featuretools/tests/entityset_tests/test_last_time_index.py

```diff
@@ -83,194 +86,194 @@ class TestLastTimeIndex(object):
             assert (pd.isnull(v1) and pd.isnull(v2)) or v1 == v2
 
     def test_leaf_no_time_index(self, es):
         es.add_last_time_indexes()
         stores = es['stores']
         true_lti = pd.Series([None for x in range(6)], dtype='datetime64[ns]')
-        assert len(true_lti) == len(stores.last_time_index)
-        stores_lti = to_pandas(stores.last_time_index)
+        assert len(true_lti) == len(stores.ww.metadata.get('last_time_index'))
+        stores_lti = to_pandas(stores.ww.metadata.get('last_time_index'))
         for v1, v2 in zip(stores_lti, true_lti):
             assert (pd.isnull(v1) and pd.isnull(v2)) or v1 == v2
 
-    # TODO: possible issue with either normalize_entity or add_last_time_indexes
+    # TODO: possible issue with either normalize_dataframe or add_last_time_indexes
     def test_parent(self, values_es, true_values_lti):
-        # test entity with time index and all instances in child entity
-        if not all(isinstance(entity.df, pd.DataFrame) for entity in values_es.entities):
-            pytest.xfail('possible issue with either normalize_entity or add_last_time_indexes')
+        # test dataframe with time index and all instances in child dataframe
+        if not all(isinstance(dataframe, pd.DataFrame) for dataframe in values_es.dataframes):
+            pytest.xfail('possible issue with either normalize_dataframe or add_last_time_indexes')
         values_es.add_last_time_indexes()
         values = values_es['values']
-        assert len(values.last_time_index) == 11
-        sorted_lti = to_pandas(values.last_time_index).sort_index()
+        assert len(values.ww.metadata.get('last_time_index')) == 11
+        sorted_lti = to_pandas(values.ww.metadata.get('last_time_index')).sort_index()
         for v1, v2 in zip(sorted_lti, true_values_lti):
             assert (pd.isnull(v1) and pd.isnull(v2)) or v1 == v2
 
     # TODO: fails with Dask, tests needs to be reworked
     def test_parent_some_missing(self, values_es, true_values_lti):
-        # test entity with time index and not all instances have children
-        if not all(isinstance(entity.df, pd.DataFrame) for entity in values_es.entities):
+        # test dataframe with time index and not all instances have children
+        if not all(isinstance(dataframe, pd.DataFrame) for dataframe in values_es.dataframes):
             pytest.xfail('fails with Dask, tests needs to be reworked')
         values = values_es['values']
 
         # add extra value instance with no children
         row_values = {'value': 21.0,
                       'value_time': pd.Timestamp("2011-04-10 11:10:02"),
-                      'values_id': 11}
+                      'values_id': 21.0}
         # make sure index doesn't have same name as column to suppress pandas warning
-        row = pd.DataFrame(row_values, index=pd.Index([11]))
-        df = values.df.append(row, sort=True)
+        row = pd.DataFrame(row_values, index=pd.Index([21]))
```

Comment:
Woodwork needs the underlying index to match the column that's set as the Woodwork index exactly. The row being added here is the 11th row, but because the value in the index column at that row is 21, the index value also needs to be 21. I can change the value in `row_values` to be 11 instead, I think, if that would make more sense.

Reply:
21 is fine 👍